main/menu.js

```diff
@@ -212,6 +212,11 @@ function buildAppMenu (options = {}) {
           accelerator: 'CmdOrCtrl+V',
           role: 'paste'
         },
+        {
+          label: l("pasteAndMatchStyle"),
+          accelerator: "", /* change this to some accelerator */
```

Comment:
This opt+shift+cmd+v on macOS, so `alt+shift+CmdOrCtrl+v"? Not sure what it is on Windows though.

Reply:
I also use Mac so I don't know, but I will look for the "alternative" for windows